pilot/cmd/pilot-agent/main.go

```diff
@@ -79,63 +72,13 @@ var (
 	podNameVar           = env.RegisterStringVar("POD_NAME", "", "")
 	podNamespaceVar      = env.RegisterStringVar("POD_NAMESPACE", "", "")
 	kubeAppProberNameVar = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
-	serviceAccountVar    = env.RegisterStringVar("SERVICE_ACCOUNT", "", "Name of service account")
```

Comment:
instanceIPvar and others have been moved to options.go? Seems redundant here?

Reply:
Good catch, i am thinking further moving all the envs and even flags out and leaves the main file clear.